doc/plugins.md

```diff
@@ -127,6 +127,8 @@ The list of plugins:
     — custom heading id support `{#custom-id}`
 *   🟢 [`remark-heading-gap`](https://github.com/remarkjs/remark-heading-gap)
     — serialize w/ more blank lines between headings
+*   🟢 [`remark-headings`](https://github.com/vcarl/remark-headings)
```

Comment:
In general this header should match the npm package name, scope and all

```suggestion
*   🟢 [`@vcarl/remark-headings`](https://github.com/vcarl/remark-headings)
```

most remark packages aren't scoped but a few like `@agentofuser/remark-oembed` are, and that is reflected in this document

Reply:
Thank you!